pylate/models/colbert.py

```diff
@@ -752,6 +754,35 @@ def pool_embeddings_hierarchical(
 
         return pooled_embeddings
 
+    @property
+    def similarity_fn_name(self) -> Literal["cosine", "dot", "euclidean", "manhattan"]:
+        """Return the name of the similarity function used by :meth:`SentenceTransformer.similarity` and :meth:`SentenceTransformer.similarity_pairwise`.
+
+        Returns:
+            Optional[str]: The name of the similarity function. Can be None if not set, in which case it will
+                default to "cosine" when first called.
+
+        Example:
```

Comment:
The docstring format won't be parsed by our doc and it could be cool to create some unit tests in this docstring